api/v1alpha1/isMonitoringCRDExists.go

```diff
@@ -0,0 +1,33 @@
+package v1alpha1
+
+import (
+	kube "k8s.io/client-go/kubernetes"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+func isMonitoringCRDExists(manager ctrl.Manager) bool {
+	config := manager.GetConfig()
+	clientset, err := kube.NewForConfig(config)
+	if err != nil {
+		databaselog.Error(err, "unable to get clientset while checking monitoring CRD")
```

Comment:
This is wrong, this error may happen on Storage or Database, but the error message will always be written into a database log. Maybe we can pass the log as a variable into this function?